entity/src/status.rs

```diff
@@ -24,3 +25,24 @@ impl Related<purl_status::Entity> for Entity {
 }
 
 impl ActiveModelBehavior for ActiveModel {}
+
+#[derive(
+    Copy,
+    Clone,
+    Eq,
+    Hash,
+    Debug,
+    PartialEq,
+    strum::EnumString,
+    strum::Display,
+    Serialize,
+    Deserialize,
+)]
+#[strum(serialize_all = "camelCase")]
```

Comment:
I think for most of the structs and enums we gave up on camelCase. Unless there's a good reason, we should stick to the default here too. I know, as a Java developer, my heart bleeds as well :grin:

Reply:
Yes, `snake_case` is actually the right one to use here.